middleware/src/handlers/handlers.go

```diff
@@ -0,0 +1,91 @@
+// Package handlers implements an api for the bitbox-wallet-app to talk to.
+package handlers
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	middleware "github.com/digitalbitbox/bitbox-base/middleware/src"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+	//"github.com/sirupsen/logrus"
```

Comment:
Nit: Could we remove commented-out import lines? (Here and elsewhere.)

Reply:
Actually, this comment is a dupe; I forgot I already mentioned this in another thread. Resolving conversation.